### PR TITLE
Update configuration-languages.md

### PR DESCRIPTION
**Pull request by salchichongallo**

### CODE REVIEW

**Comment by montogeek** [comment]
Flag option changed? Or it was jurt a typo?

**Comment by montogeek** [comment]
@salchichongallo Please sign CLA.

File: src/content/configuration/configuration-languages.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ __tsconfig-for-webpack-config.json__
 
 T> `ts-node` can resolve a `tsconfig.json` file using the environment variable provided by `tsconfig-path`.
 
-Then set the environment variable `process.env.TS_NODE_PROJECT` provided by `tsconfig-path` like so:
+Then set the environment variable `process.env.TS_NODE_PROJECT` provided by `tsconfig-paths` like so:
```
Comment (asapach) [inline src/content/configuration/configuration-languages.md]: it also says `tsconfig-path` on the line above

**Comment by montogeek** [comment]
@salchichongallo Friendly ping

**Comment by montogeek** [comment]
Original fork doesn't exist anymore.

**Pull request closed without merging**